pages/common/monop.md

```diff
@@ -0,0 +1,31 @@
+# monop
+
+> Finds and displays signatures of types and methods inside .NET assemblies.
+
+- Show the structure of a type built-in to the .NET Framework:
```

Comment:
of a built-in type of the .NET framework ? (Should the F be capital or small ?)

Reply:
Thanks! And it should be a capital, as that's it's the name of a thing and the way it appears on their website.

Reply:
I actually wanted _ built-in type of_

Reply:
It just reads natural to me. For eg. "int, and char are built-in types of the C programming language" sounds better than "int and char are types built-in to the C programming language.".

But up to you.

Reply:
I see. Perhaps I should make `type` uppercase then. It's a specific class that holds information about a data-type, and is used in reflection. This command lists all such Types.

Reply:
Sure.